CSS folder/index.css

```diff
@@ -0,0 +1,269 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Times New Roman", Times, serif;
+  line-height: 1.4;
+}
+
+a {
+  text-decoration: none;
+}
+
+p {
+  margin: 0.75rem 0;
+}
+.text-primary {
+  color: #ffa500;
+}
+
+.navbar {
+  display: flex;
+  background: #333;
+  color: #fff;
+  position: sticky;
+  top: 0;
+  justify-content: space-between;
+  z-index: 1;
+  padding: 16px;
+}
+.navbar ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.navbar ul li a {
+  color: #fff;
+  padding: 0.75rem;
+  margin: 0 0.25rem;
+}
+
+.navbar ul li a:hover {
+  background: #ffa500;
+  border-radius: 5px;
+}
+
+/* Header*/
+.headers {
+  background: #333 url("../images/office-computer.jpg") no-repeat center
```

Comment:
What is this line actually doing for you? Kindly confirm if these values had an effect on your site

Reply:
Yes, it does. It is a background image for the site